3.2/docker-entrypoint.sh

```diff
@@ -28,11 +28,14 @@ case "$1" in
 		if [ ! -f './config/database.yml' ]; then
 			file_env 'REDMINE_DB_MYSQL'
 			file_env 'REDMINE_DB_POSTGRES'
+			file_env 'REDMINE_DB_SQLSERVER'
 			
 			if [ "$MYSQL_PORT_3306_TCP" ] && [ -z "$REDMINE_DB_MYSQL" ]; then
 				export REDMINE_DB_MYSQL='mysql'
 			elif [ "$POSTGRES_PORT_5432_TCP" ] && [ -z "$REDMINE_DB_POSTGRES" ]; then
 				export REDMINE_DB_POSTGRES='postgres'
+			elif [ "$SQLSERVER_PORT_1433_TCP" ] && [ -z "$REDMINE_DB_SQLSERVER" ]; then
+				export REDMINE_DB_SQLSERVER='sqlserver'
```

Comment:
Should also drop these two lines, since they are also about a linked container.

Reply:
Will do. But can you help me to understand how this is bad to include for SQL server, but good to include for PostgreSQL and MySQL?

Reply:
They are still there for backwards compatibility to use the presence of environment variables to detect a `--link`ed container. Docker [docs](https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/) call links a legacy feature that "may eventually be removed".  We should instead point users to an example compose/stack file that sets all of the correct environment variables.